src/js/game/hud/parts/building_placer_logic.js

```diff
@@ -114,6 +120,34 @@ export class HUDBuildingPlacerLogic extends BaseHUDPart {
         this.root.camera.upPostHandler.add(this.onMouseUp, this);
     }
 
+    /**
+     * Returns the current base rotation for the current meta-building.
+     * @returns {number}
+     */
+    get currentBaseRotation() {
+        const rotationByBuilding = this.root.app.settings.getAllSettings().rotationByBuilding;
+        if (!rotationByBuilding) {
```

Comment:
I think it'd be fine to put this into one line without a variable

Reply:
Moved both instances of this to one-liners.